corehq/apps/api/odata/tests/data/empty_metadata_document.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes" ?>
+<edmx:Edmx xmlns:edmx="http://docs.oasis-open.org/odata/ns/edmx" Version="4.0">
+    <edmx:DataServices>
+        <Schema xmlns="http://docs.oasis-open.org/odata/ns/edm" Namespace="CommCare">
+            <EntityContainer Name="Container" >
+            </EntityContainer>
```

Comment:
realizing this is just exposing existing behavior (which is great! hooray tests!) but it seems a little odd to me that we wouldn't include the built in meta properties even if you have no custom properties defined. is that by design?

Reply:
The point of this test is for when there are no case types, so we wouldn't include the meta properties, I think.

Reply:
oh whoops sorry i completely misread the test.